src/app/app.module.ts

```diff
@@ -55,6 +55,8 @@ import { DateValidatorDirective } from './core/directives/dateValidator.directiv
 
 import { AngularFireModule } from 'angularfire2';
 import { Ng2BreadcrumbModule, BreadcrumbService } from 'ng2-breadcrumb/ng2-breadcrumb';
+import { StoreModule } from '@ngrx/store';
+import { counterReducer } from './core/services/login.service';
```

Comment:
myśle, ze to nie jest najlepsze miejsce dla reducera `./core/services/login.service`